pyspecProcScripts/correlation_alignment.py

```diff
@@ -172,7 +158,11 @@ def correl_align(
     energy_diff = 1.0
     i = 0
     energy_vals = []
-    this_E = (abs(s_orig.C.sum(indirect_dim)) ** 2).data.sum().item() / N**2
+    # below we divide by an extra N because if the signals along the repeats
+    # are the same, then the energy of the resulting sum should increase by N
+    # (vs taking the square and summing which is what we do for calculating the
+    # sig_energy above)
+    this_E = (abs(s_jk.C.sum("repeats")) ** 2).data.sum().item() / N**2
     energy_vals.append(this_E / sig_energy)
     last_E = None
     # TODO ☐: the mask needs to be separated into
```

Comment:
this is the comment you were talking about earlier in our conversation where you basically lay the ground work for the next PR which makes the mask a function